### DISCUSSION TITLE
"What is being produced in this orchard" question

### DISCUSSION THREAD

**Discussion by okainov** [streetcomplete/StreetComplete#5692] [General]
I read about the history of this question here https://github.com/streetcomplete/StreetComplete/issues/368. I read OSM wiki about `orchard` https://wiki.openstreetmap.org/wiki/Tag:landuse=orchard. I see that initially in SC the tags were added as `crop` and/or `trees`. This makes perfect sense for me - we try to map what is growing on this piece of land (orchard).

However what leaves me confused in this commit https://github.com/streetcomplete/StreetComplete/commit/eb8d361e63ecfdbd4001a88379ac1d5014aa0326 which has updated `trees`/`crop` to `produce`. OSM wiki says `The tag is particularly helpful in cases where the type of trees is not descriptive enough`, but normally it is descriptive enough, right?

Main question is why? What was wrong with `trees`/`crop`? What is the benefit/reasoning behind using `produce` here? I think of `produce` more like what is being sold at the farm shop/stall/kiosk next to the orchard, so for me it would be more suitable tag for the kiosk/shopping point rather than for the field itself...

**Comment by matkoniecz** [comment]
Has wiki maybe changed since it was edited 7 years ago?

**Reply by okainov** [reply]
Maybe (though I personally doubt here). 

But still, would be great to know the intent of the change, as I think it was actually correct to use `crop`/`trees` in the first place.

**Reply by okainov** [reply]
BTW here is some stats in particular about strawberries, `crop` is way more popular than `produce` (though of course, it also depends on the objects themselves)

![image](https://github.com/streetcomplete/StreetComplete/assets/918446/073789b8-b7bb-428f-82e1-ab9950e552bd)

**Reply by mnalis** [reply]
> But still, would be great to know the intent of the change, 

`crop` does seem more reasonable (and more popular) choice for me too, but there were some issues with it (e.g. https://github.com/streetcomplete/StreetComplete/issues/368#issuecomment-312498580). Pinging @rugk as he was very involved in original quest, so maybe he have some ideas / input why `produce=*` is used nowadays instead of `trees=*`.

> as I think it was actually correct to use `crop`/`trees` in the first place.

Don't know if it is related, but it seems that `OsmItem` was simplified to only contain the `value` (instead of `key=value`) at the time too, so that would mean only one key could be used.

**Reply by rugk** [reply]
Thx for the ping, unfortunately I was not involved so much with the change to `produce` and also dunno…

But you can certainly find related discussion about this one:
* [Proposal to avoid produce=*](https://wiki.openstreetmap.org/wiki/Talk:Tag:landuse%3Dorchard#Proposal_to_avoid_produce=*)
* Here e.g. much about produce has been changed/done: https://wiki.openstreetmap.org/w/index.php?title=Tag%3Alanduse%3Dorchard&type=revision&diff=1911545&oldid=1892596
* wiki talk disc. about bad combinations with produce? https://wiki.openstreetmap.org/wiki/Talk:Tag:landuse%3Dorchard#Removal_of_useful_combinations
* on the trees wikitalk page, one argues `produce` should be preferred: https://wiki.openstreetmap.org/wiki/Talk:Key:trees#A_preference_for_the_key_produce_rather_than_trees.
